libs/hdf-converters/src/ckl-mapper/checklist-jsonix-converter.ts

```diff
@@ -22,6 +22,7 @@ import {
   Vulnattribute
 } from './checklistJsonix';
 import {coerce} from 'semver';
+import {validateChecklistProfileMetadata} from './checklist-mapper';
```

Comment:
Do not make the imports circular.  Either move everything here (if it makes sense) or put it into its own file.

Reply:
Done